src/config/config.go

```diff
@@ -49,6 +49,9 @@ var (
 	// CLIVersion track the version of the CLI
 	CLIVersion = "unset"
 
+	// GlobalOptions tracks user-defined values that apply across commands
+	GlobalOptions types.ZarfGlobalOptions
```

Comment:
I don't _love_ `GlobalOptions`, how much would you hate `CLIOptions`, `UserOptions`, `UserFlags`, anything besides `GlobalOptions`?  😆 I also don't have a super compelling reason, just global feels weird.

Reply:
I do like how the common options are now separate from the workflow-specific options

Reply:
CommonOptions works for me, thanks!